server/database.js

```diff
@@ -0,0 +1,35 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  database: 'product_db',
+});
+
+const getProduct = function getProductInformation(productId, callback) {
+  connection.query(`select * from products where id=${productId}`, (err, results) => {
+    if (err) console.error(err);
+    callback(results);
+  });
+};
+
+const getRelated = function getRelatedProducts(productName, productId, callback) {
+  connection.query(`select id, product_tier, price, stock_count, thumbnail from products where name='${productName}' and id <> ${productId}`, (err, results) => {
```

Comment:
What does the '<>' do here?   Is this a typo?

Reply:
On second thought, is this trying to grab every product that doesn't have that exact productId?

Reply:
Yes, this is analogous to '!=='/does not equal.